src/renderer/scss/_gui.scss

```diff
@@ -188,3 +188,23 @@ section.section-spaced {
 {
   text-align: center;
 }
+
+.error-text {
+  color: var(--color-error);
+}
+
+.display-inline-block {
```

Comment:
IMO this and all of the definitions below it probably shouldn't exist.

Reply:
Not a fan of these? I think they can be pretty helpful to avoid creating a brand new class just to add a simple display style.

Totally down to get rid of them if you don't want them.